tests/widget/modal/widget-modal.login.test.js

```diff
@@ -5,6 +5,13 @@ import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
 test('Modal widget with login', async ({ page }) => {
   const { clickButton, navigate } = asyncEvents(page);
 
+  // Navigate to page without Widget instantiation
+  await navigate('widget');
+
+  const loginLink = page.getByRole('link', { name: 'Login via Modal Widget' });
+
+  await expect(loginLink).toBeVisible();
+
```

Comment:
Added a bit more to this basic test to ensure the misc APIs work before and after logging in.